packages/wrangler/src/config/validation.ts

```diff
@@ -194,6 +202,10 @@ export function normalizeAndValidateConfig(
 	// Process the top-level default environment configuration.
 	const config: Config = {
 		configPath,
+		pages_build_output_dir: normalizeAndValidatePagesBuildOutputDir(
```

Comment:
I tried to keep all Pages things as contained as possible, with the the idea of the converged future in mind. However, this bit IMO, needs to happen here